napalm_junos/junos.py

```diff
@@ -204,8 +206,13 @@ def _load_candidate(self, filename, config, overwrite):
 
             if fmt == "xml":
                 configuration = etree.XML(configuration)
-
-            self.device.cu.load(configuration, format=fmt, overwrite=overwrite)
+            # add ignore_warning optional arg
+            if self.ignore_warning:
```

Comment:
Ok, the PR is good but the if..else is not really necessary. You can just write it as:

```
    self.device.cu.load(configuration, format=fmt, overwrite=overwrite,
                        ignore_warning=self.ignore_warning)
```